v1/rest/rest.go

```diff
@@ -265,8 +265,9 @@ func PublishLog(context *gin.Context, status int, payload interface{}, msg ...st
 		return nil
 	}
 
+	contentType := context.GetHeader("Content-Type")
 	var requestBodyInterface map[string]interface{}
-	if len(requestBody) > 1 {
+	if contentType == "application/json" && len(requestBody) > 1 {
```

Comment:
```suggestion
	if contentType == "application/json" && len(requestBody) > 0 {
```

pak titip tolong ubah dong